recipe/meta.yaml

```diff
@@ -24,8 +24,8 @@ requirements:
   - bob.io.base
   - giflib
   - libpng
```

Comment:
Technically we should probably be pinning libpng too. Haven't figured out which version yet though...